packages/cli/src/commands/new/project.ts

```diff
@@ -58,6 +58,10 @@ export default class Project extends Command {
       description: 'skip git initialization',
       default: false,
     }),
+    interactive: flags.boolean({
+      description: 'choose options rather than defaults',
```

Comment:
Maybe...?
```suggestion
      description: 'run an interactive wizard to configure your project',
```

Reply:
yes this makes more sense. my original description was quite literal haha